src/starbase/setup.ts

```diff
@@ -14,6 +21,32 @@ async function installIntegrationDependencies(directory: string) {
   return executeWithLogging(`yarn --cwd ${directory} install`);
 }
 
+/**
+ * After we have cloned the repository, pull down information on the required
+ * instanceConfigFields and perform additional checks.
+ */
+async function checkInstanceConfigFields(integration: StarbaseIntegration) {
```

Comment:
Is there ever a case where we want to ignore this check? I cant necessarily think of one off hand but curious if you think theres value in giving the ability to ignore the check/validation.